src/application/utils/licence-type.ts

```diff
@@ -0,0 +1,21 @@
+const MANUAL: string = 'Manual';
+const AUTOMATIC: string = 'Automatic';
+
+export function licenceToIssue(
+    category: string,
+    vehicleTransmission: string,
+    code78Present?: boolean): string {
+
+  const categoriesToCheck: string[] = ['C', 'D', 'C+E', 'D+E'];
+
+  if (code78Present === undefined) {
```

Comment:
Do you need this `if` condition? Seems the only time you would return something else is handled by the second `if` block below.

Reply:
I think it's needed as it could potentially be considered falsey and incorrectly get assessed in the logic below.